airbyte-webapp/src/components/connectorBuilder/Builder/BuilderSidebar.tsx

```diff
@@ -25,6 +25,7 @@ interface ViewSelectButtonProps {
   selected: boolean;
   showErrorIndicator: boolean;
   onClick: () => void;
+  testId: string;
```

Comment:
It's named differently because it gets a prefix automatically while the others are "raw" `data-testid`. However this is probably unexpected and too much magic, so I flattened out the prefix in the places where `ViewSelectButton` is actually used. A bit more redundancy but it's probably for the better.